HelpSource/Classes/Function.schelp

```diff
@@ -510,7 +510,8 @@ argument::bufsize
 The size of the buffer for the ScopeView. The default is 4096.
 argument::zoom
 A zoom value for the scope's X axis. Larger values show more. The default is 1.
-
+argument:: bounds
```

Comment:
maybe you don't need to add this long explanation, because the behaviour of the scope view may change. Maybe just refer to the documenattion of the scope view.

Reply:
Thank you for that. 

However, `ScopeView` has no corresponding part in the documentation and `link::Classes/View#*new::` is linked from the inherited class methods.
Also, the `bounds' argument is related to the bounds method of the `Stethoscope'.

Which of the following should be placed in line 514?

1. `The bounds of the scope window. The default is nil and the scope window will be in the centre of the display. See link::Classes/Stethoscope#-bounds::`
2. `The bounds of the scope window. The default is nil and the scope window will be in the centre of the display. See link::Classes/View#*new::`

Reply:
It is the bounds of the stethoscope, right?

Reply:
Yes. 

I originally proposed it as an argument of the Stethoscope class: 
    #6230 

Then @mtmccrea suggested it as a method: 
     https://github.com/supercollider/supercollider/pull/6230#issuecomment-2068195977

I agreed: 
    https://github.com/supercollider/supercollider/pull/6230#issuecomment-2068293011

Reply:
>1. The bounds of the scope window. The default is nil and the scope window will be in the centre of the display. See link::Classes/Stethoscope#-bounds::

This is probably most appropriate because it calls `Stethoscope:-bounds`

It may clarify more to write:|
"The bounds of the scope code::Window::."

This would disambiguate it from the embedded ScopeView.